Subtester/Packages/packages-lock.json

```diff
@@ -71,6 +71,13 @@
       "dependencies": {},
       "url": "https://packages.unity.com"
     },
+    "com.unity.nuget.newtonsoft-json": {
```

Comment:
What's this dependency for?

Reply:
I can confidently say that I have no idea 😂 
I can revert it, [from the docs](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@2.0/manual/index.html) it looks like it's an internal unity thing